src/__tests__/fields/unit-number-field.spec.ts

```diff
@@ -0,0 +1,40 @@
+import { jsonToSchema } from "../../schema-generator";
+import { ERROR_MESSAGES } from "../../shared";
+import { TestHelper } from "../../utils";
+import { ERROR_MESSAGE } from "../common";
+
+describe("unit-number-field", () => {
+	it("should be able to generate a validation schema", () => {
+		const schema = jsonToSchema({
+			section: {
+				uiType: "section",
+				children: {
+					field: {
+						uiType: "unit-number-field",
+						somethingUnused: "test",
+						validation: [{ required: true, errorMessage: ERROR_MESSAGE }],
+					},
+				},
+			},
+		});
+		expect(() => schema.validateSync({ field: "01-19" })).not.toThrowError();
+		expect(TestHelper.getError(() => schema.validateSync({})).message).toBe(ERROR_MESSAGE);
+	});
+
+	it("should display invalid unit number error message", () => {
```

Comment:
the library does not display anything technically, should avoid using `display` in test description